### DISCUSSION TITLE
Does Micrsoft.Ideneity.Web clean up token cache after user signout? (with calldownstreamAPI enabled)

### DISCUSSION THREAD

**Discussion by felixt-c** [AzureAD/microsoft-identity-web#1292] [Q&A] [answered]
Also, how about the offline_access enabled scenerio?

**Comment by jmprieur** [comment, marked as answer]
yes @llx9, it does clear the cache when the user signs-out.

**Reply by felixt-c** [reply]
Hi @jmprieur How about the "offline_access" scope. Does Micrsoft.Ideneity.Web automatically request "offline_access" scope as part of OpenID Connect on user sign in so that developer does not need to sepify this scope seperately inisde EnableCallDownStreamAPI?

**Reply by jmprieur** [reply]
Yes, @llx9 Microsoft.Identity.Web requests offline_access, openid, and profile.

**Reply by felixt-c** [reply]
thank you for the answer.
